### ISSUE TITLE
Resizing charts in study view

### ISSUE THREAD

**Issue by jjgao** [cBioPortal/cbioportal#7293]
- [x] resizing ui
- [x] save to user preference of the study layout after resizing

@zhx828 I thought we had an issue about this but couldn't find it.

**Comment by zhx828** [comment]
@jjgao we briefly mentioned here https://github.com/cBioPortal/cbioportal/issues/4460
Also there is a demo in https://github.com/cBioPortal/cbioportal-frontend/pull/1298 but we decided to not enable that.

**Comment by jjgao** [comment]
@schultzn @tmazor any thoughts about cBioPortal/cbioportal-frontend#1298. I think we/I didn't like the small icon on the bottom-right corner and thought it would be ideal if it is only shown when hovering. But now I think it would be nice to have the resizing function even if we have to live with the small icon.

![jul-23-2018 15-40-12](https://user-images.githubusercontent.com/5400599/43098982-b700c8fa-8e8e-11e8-8bf9-d74ebe8ad0af.gif)

**Comment by schultzn** [comment]
Yes, it looks ok to show it for every chart.

**Comment by tmazor** [comment]
Agree, I think it's ok to have that icon on all charts

**Comment by jjgao** [comment]
Great. @zhx828 could you revive cBioPortal/cbioportal-frontend#1298? Sorry for the back and forth.

**Comment by jjgao** [comment]
@zhx828: @Luke-Sikina is interested in giving this one a try.

**Comment by zhx828** [comment]
@jjgao awesome. @Luke-Sikina keep me posted if you need any help. It's been a while since this ticket was created. But I'm happy to go through the details with you.

**Comment by jjgao** [comment]
@Luke-Sikina what's left before we can merge?
